### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by MarcelBochtler**
See commit messages

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -169,4 +169,4 @@ xz = { module = "org.tukaani:xz", version.ref = "xz" }
 exposed = ["exposedCore", "exposedDao", "exposedJdbc", "exposedJavaTime"]
 hoplite = ["hopliteCore", "hopliteHocon"]
 maven = ["mavenCore", "mavenCompat"]
-mavenResolver = ["mavenResolverConnectorBasic", "mavenResolverTransportFile", "mavenResolverTransportHttp", "mavenResolverTransportWagon"]
```
Comment (sschuberth) [inline gradle/libs.versions.toml:172]: This change seems unrelated.

**Review by sschuberth — COMMENTED** [review]
"Exposed" should be capitalized in the commit message.

File: gradle/libs.versions.toml (hunk 1 above)
Comment (MarcelBochtler) [reply gradle/libs.versions.toml:172]: Moved it to a separate commit.

**Review by sschuberth — APPROVED** [review]
